pyproject.toml

```diff
@@ -58,3 +58,5 @@ documentation = "https://stravalib.readthedocs.io"
 repository = "https://github.com/stravalib/stravalib"
 changelog = "https://github.com/stravalib/stravalib/blob/master/changelog.md"
 
+[tool.black]
+line-length = 79
```

Comment:
Why do we deviate from the default black line length?

Reply:
Great question - i've updated the docs with this.
What i was trying to explain in that issue about black vs blue. Black makes some decisions that some people do not like. [The actual PEP 8 spec is 79 characters ](https://peps.python.org/pep-0008/#maximum-line-length). Most change black to 79. but blue (which wraps around black) defaults to 79. it's just a quirk of black that some like and others do not like :) i prefer to stick to the PEPs whenever possible

Reply:
Ok, that makes sense. Just as a reminder: I had to add the same (explicit) black config to the [swagger2pydantic](https://github.com/stravalib/strava_swagger2pydantic) repo to prevent apparent diffs between the existing and generated pydantic model file. So whenever we choose to change any black config in this repo, it should also be changed in the other manually.

Maybe to be sure someone in the future will not accidentally mess up, we can add this as a comment above the black config? (unfortunately, Github doesn't allow me to make a suggestion anymore for this conversation, the suggested line is: `# When editing black configuration, make sure to make the same edits in the repo stravalib/strava_swagger2pydantic`)

Reply:
Got it! I've added a note in the pyproject.toml file. I think you couldn't make the edit here because the file had changed (i hope anyway) and you need to then make a new comment on the new version of the file in the "files" list on GitHub? maybe? but also comments have been quirky for me lately on GitHub so it could be that too! i've made the change and will merge this! thanks for all of the feedback! i will be adding a few more format checkers in the upcoming weeks that focus on documentation :) but i'll open separate issues and PR's for those of course